keymap/standard.go

```diff
@@ -60,6 +60,7 @@ var StandardMaps = Maps{
 		"Backspace":           Backspace,
 		"Control+Backspace":   BackspaceWord,
 		"Alt+Backspace":       BackspaceWord,
+		"Meta+Backspace":      BackspaceWord,
```

Comment:
Cmd+Backspace is supposed to delete to the start of the line, not the start of the word. Also, the same is true for Cmd+Delete.

Reply:
Actually, it looks like Cmd+Delete isn't generally supported, so we just need Cmd+Backspace to work for delete line.

Reply:
We do not have a Function for delete to start of line. You can file an issue for that, but it is not going to happen in this PR. 

Also, clarify Cmd+Delete: is that the same as delete-to-end-of-line?  That would be more consistent with "forward delete" vs. "backward delete". We DO already have delete-to-end-of line: Kill -- that is easy.

Reply:
oh right you said no Cmd+Delete already?  where id that comment go?

Reply:
Theoretically:

* Cmd+Backspace: delete to start of line
* Cmd+Delete: delete to end of line

In practice, it seems like other apps support Cmd+Backspace but not Cmd+Delete. I use Cmd+Backspace a lot. There is no existing keymap for the inverse of Kill?

Reply:
nope. again, file an issue and approve this. and LMK if you want to go ahead and map Cmd+Delete to Kill.

Also for cross-platform consistency, what is the non-Meta (Cmd) version of Cmd+Delete = Kill for other platforms? Alt? Control?  which one goes to word and which one goes to line?

looking at: https://github.com/cogentcore/core/discussions/650 -- it seems like Control+ is word in general. So it would overall be more consistent to have Control+nav = word; Alt+nav = line. and on mac, Meta = Cmd works just like Control.

again probably better to sort this out in another issue, and go ahead and merge this.